source/includes/_registry.md

```diff
@@ -0,0 +1,106 @@
+# Registry
+
+The registry is the low-level host functionality that provides the foundation
+for [SkyDB](#skydb). It allows getting and setting registry entries with a
+user's key and a data key (e.g. the name of an app).
+
+Registry entries contain data, the data key, and a revision number. The revision
+number is incremented by 1 every time the registry entry is changed. The latest
+revision number, plus one, is required when running `setEntry`. Because data at
+a given revision number cannot be modified, the registry effectively provides
+versioned, immutable data.
+
+<aside class="warning">
+These functions have only been implemented for Browser JS at the moment.
+</aside>
+
+## Getting Data From The Registry
+
+```javascript--browser
+import { SkynetClient, keyPairFromSeed } from "skynet-js";
+
+const client = new SkynetClient();
+const { publicKey, privateKey } = keyPairFromSeed("this seed should be fairly long for security");
+
+const dataKey = "myApp";
+
+async function getEntryExample() {
+  try {
+    const { entry, signature } = await client.registry.getEntry(publicKey, dataKey);
+  } catch (error) {
+    console.log(error);
+  }
+}
+```
+
+### Function
+
+`getEntry`
+
+### Parameters
+
+Field | Type | Description
+----- | ---- | -----------
+`publicKey` | `Buffer` or `Uint8Array` | User's public key. Can be generated with the `keyPairFromSeed` function.
+`dataKey` | `string` | The key of the data to fetch for the given user.
+
+### Response
+
+```javascript--browser
+{
+  entry: {
+    datakey: "HelloWorld",
+    data: "abc",
+    revision: 123456789
```

Comment:
let's use basic `"foo": "bar"` in all examples and a revision number that's closer to what the user will see when using the api (I would just put `0` here)